language-design.md

```diff
@@ -16,8 +16,22 @@ are:
 For each quote, describe what it was about the quote that led you pick it.
 
 **Response**
+"We have to do this a lot when we write real computer programs: a thought that seems like a primitive in our minds turns out not to be a primitive in a programming language, and in each new program we must define it once more." [Steele, 1998]
 
+I chose this quote because I thought it was epecially true and interesting. It highlights the differences in various programming languages. Perhaps if you take some primitives to be more important than others then you will perfer languages with those defined primitives. -Daniel
```

Comment:
I agree with the sentiment of the quote as well. There have been several times after coding extensively in python that I find myself using primitives that simply do not exist (like what Aaron said below with max / min).